tests/projects/test_projects.py

```diff
@@ -133,8 +133,8 @@ def test_run_local_git_repo(local_git_repo, local_git_repo_uri, use_start_run, v
     # Validate run contents in the FileStore
     run_id = submitted_run.run_id
     mlflow_service = MlflowClient()
-    run_infos = mlflow_service.list_run_infos(
-        experiment_id=FileStore.DEFAULT_EXPERIMENT_ID, run_view_type=ViewType.ACTIVE_ONLY
+    run_infos = mlflow_service.search_runs(
+        [FileStore.DEFAULT_EXPERIMENT_ID], run_view_type=ViewType.ACTIVE_ONLY
     )
     assert len(run_infos) == 1
     store_run_id = run_infos[0].run_id
```

Comment:
Should these accessors be `run_infos[0].info.run_id`?

Reply:
Yep! thanks Ben!